code/game/machinery/washing_machine.dm

```diff
@@ -162,8 +162,7 @@
 				icon_state = initial(G.icon_state)
 				item_color = wash_color
 				name = "washed gloves"
-				desc = "The colors are a bit dodgy."
-				break
+				desc = "The colors are a bit dodgy." // removed break here, it should use the last item in the list for its color. See black gloves.
```

Comment:
```suggestion
				desc = "The colors are a bit dodgy." 
```
Don't take this 100% as gospel truth, but you probably don't need this comment here. Nobody needs to know specifically that a break was removed, that's what github history is for. This info wont actually be useful to anyone reading the code, right? I dunno.